Administrator Guide/Thin-Arbiter-Volumes.md

```diff
@@ -0,0 +1,72 @@
+# Thin Arbiter volumes in gluster
+
+Thin Arbiter is a new type of quorum node where granularity
+of data is less compared to the traditional arbiter brick.
+In this type of volume, quorum is taken into account at a brick
+level rather then per file basis. If there is even one file/directory
+which is source for IO on a particular brick, the other brick will
+not be allowed to become source for any other file/directory.
+
+<!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Thin Arbiter Configuration](#thin-arbiter-configuration)
+- [Why Thin Arbiter?](#why-thin-arbiter)
```

Comment:
Modifying the order and also changing it to "Setting UP Thin Arbiter Volume" instead of "Using thin arbiter" as this section contains only commands to setup TA volume.